sql/dj_setup_database.in

```diff
@@ -234,10 +242,15 @@ install_examples()
 	( cd "$EXAMPLEPROBDIR" && yes y | "$BINDIR"/import-contest )
 }
 
+create_database_dump () {
+	sudo mysqldump $(mysql_options) --opt --skip-lock-tables domjudge | pv | gzip > "$DATABASEDUMPDIR/${1}.sql.gz"
+	exit 0
```

Comment:
This is a weird side effect of the function

Reply:
Not doing it makes the code annoying to read, but I agree the side effect is weird.